assets/l10n/en-US.ftl

```diff
@@ -451,7 +451,17 @@ fcm_group_title =
         [yes] ...
        *[no] {""}
     }
-fcm_incoming_call = Incoming call
+fcm_incoming_call =
+    { $type ->
+        [dialog] Incoming
+        *[group] {$userName ->
+           [x] {$userNum}
+           *[other] {$userName}
+        } started a
+    } { $isVideo ->
+        [yes] video
+        *[other] audio
```

Comment:
> Shouldn't we determine the default variables the way that won't break the current phrases?

Fixed these two phrases, but I think the other should be refactored too (For example `fcm_message` has the same problem).

Reply:
@50U10FCA7, I was just concerned regarding the BCs introduced with the `fcm_incoming_call` phrase. With the current backend version it'll break the incoming call push notification when this PR is merged, since backend doesn't send any parameters when forming it. The mentioned `fcm_message` won't break, since the current backend version sends all the parameters required for the phrase to be displayed correctly.

Reply:
@SleepySquash 

>  I was just concerned regarding the BCs introduced with the fcm_incoming_call phrase. With the current backend version it'll break the incoming call push notification when this PR is merged, since backend doesn't send any parameters when forming it. 

`[ResolverError(Reference(Variable { id: "type" })), ResolverError(Reference(Variable { id: "isVideo" }))]`

Still BC 🙃 

Fluent doesn't fill non-passed vars with nulls/empty strings.